deploy/examples/create-external-cluster-resources.py

```diff
@@ -1829,6 +1855,32 @@ def gen_json_out(self):
                     }
                 )
 
+        # if 'TOPOLOGY_POOLS', 'TOPOLOGY_FAILURE_DOMAIN_LABEL', 'TOPOLOGY_FAILURE_DOMAIN_VALUES'  exists,
+        # then only add 'topology' StorageClass
+        if (
+            self.out_map["TOPOLOGY_POOLS"]
+            and self.out_map["TOPOLOGY_FAILURE_DOMAIN_LABEL"]
+            and self.out_map["TOPOLOGY_FAILURE_DOMAIN_VALUES"]
+        ):
+            json_out.append(
+                {
+                    "name": "ceph-rbd-topology-storageclass",
```

Comment:
How about an optional argument for the storage class name? This is a good default, but users commonly want to customize it.

Reply:
lets discuss

Reply:
As discussed, let's not add an argument to keep it consistent with other storage classes generated for external clusters.